CIPs/cip-0021.md

```diff
@@ -82,7 +83,7 @@ def get_lookback_window(chain: ChainState) -> int:
         window = chain.evm.get_blockchain_param("lookbackWindow")
```

Comment:
github won't let me comment on the right line.  But the comment above: `# default to 12 in case of error` needs to be changed to reflect the changes.  Also this function is missing the `epochSize - 2` part